node/buffer_test.ts

```diff
@@ -298,6 +298,32 @@ Deno.test({
   },
 });
 
+Deno.test({
+  name: "Buffer readUIntBE",
+  fn() {
+    const buffer = Buffer.from([
+      0x01,
+      0x02,
+      0x03,
+      0x04,
+      0x05,
+      0x06,
+      0x07,
+      0x08,
+    ]);
+    assertEquals(buffer.readUIntBE(0, 1), 0x01);
+    assertEquals(buffer.readUIntBE(0, 2), 0x0102);
+    assertEquals(buffer.readUIntBE(0, 4), 0x01020304);
+    assertThrows(
+      () => {
+        assertEquals(buffer.readUIntBE(0, 5), 0x01020304);
+      },
+      Error,
+      `The value of "byteLength" is out of range. It must be >= 1 and <= 4. Received 5`,
```

Comment:
I think the error message should say `The value of "byteLength" is out of range. It must be >= 1 and <= 6. Received ...` because it actually supports 1 to 6, and we should throw `notImplemented` errors in the case of byteLength is 3, 5, or 6